src/conditionals/programs_with_decision_points/hard/q7/MainTest.java

```diff
@@ -0,0 +1,69 @@
+package conditionals.programs_with_decision_points.hard.q7;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+public class MainTest extends BaseTest {
+  // Parsons with distractors
+  public Clause[] testSentence() {
+    TestOption.isInputTest = true;
+    TestOption.defaultInput = "1000";
+    return new Clause[] {
+        new StringLiteral("How many people are in your area\\?"),
+        new NewLine(),
+        new PlaceHolder()
+    };
+  }
+
+  public void runMain() {
+    RiskyBusiness.main(new String[0]);
+  }
+
+  static Stream<Arguments> inputProviderValid() {
+    return Stream.of(
+        Arguments.of(1, "Low Risk"),
+        Arguments.of(100, "Low Risk"),
+        Arguments.of(999, "Low Risk"),
+        Arguments.of(1000, "Moderate Risk"),
+        Arguments.of(5000, "Moderate Risk"),
```

Comment:
I would try to not always test the exact middle number, something more random is less predictable (maybe 6739 for example).